peachjam/admin.py

```diff
@@ -613,8 +613,14 @@ class CoreDocumentAdmin(DocumentAdmin):
     pass
 
 
+class AttachedFilesInline(BaseAttachmentFileInline):
+    model = AttachedFiles
+    form = AttachedFilesForm
+
+
 class GenericDocumentAdmin(ImportExportMixin, DocumentAdmin):
     resource_class = GenericDocumentResource
+    inlines = [AttachedFilesInline] + DocumentAdmin.inlines
```

Comment:
Can't we put the attached files inline into DocumentADmin.inlines? The attachments apply to all documents.